domain/machine/service/service.go

```diff
@@ -53,6 +53,16 @@ type State interface {
 	// DeleteMachineCloudInstance removes an entry in the machine cloud instance table
 	// along with the instance tags and the link to a lxd profile if any.
 	DeleteMachineCloudInstance(context.Context, string) error
+
+	// RequireMachineReboot sets the reboot flag for the machine referenced by its UUID, which indicates that the
```

Comment:
Let's change `flag` to row. As the flag indicates it's a column with a boolean value.

Reply:
I will do better, i will remove the term `flag` from `service` layer to not leak implementation in its comments.

Regarding services, the fact that reboot is handled by flag or by row has no meaning.

However, I will make sure that `state` layer don't use the word "flag" but use `row` instead.